build/nuget/build.ps1

```diff
@@ -0,0 +1,32 @@
+<#
+.SYNOPSIS
+    Creates a nuget package for the specified nuspec.
+#>
+Param (
+        [Parameter(Mandatory=$true, HelpMessage="nuspec file")]
+        [string]$Nuspec,
+
+        [Parameter(Mandatory=$false, HelpMessage="repo to pull binaries for nuspec")]
+        [string]$Repo,
+
+        [Parameter(Mandatory=$false, HelpMessage="Build Configuration")]
+        [ValidateSet('debug', 'release')]
+        [string]$BuildConfiguration = 'debug'
+)
+
+$scriptPath = (get-item $MyInvocation.MyCommand.Path).Directory.FullName
+if(!$Repo) {
+    $Repo = [System.IO.Path]::Combine($scriptPath, "..\..")
+}
+
+$version = "0.0.1"
+$timeStamp = get-date -format yyyyMMdd-HHmm
```

Comment:
.0.0.1-20161117113815.pr.nupkg

---
In reply to: [88534964](https://github.com/Microsoft/WinObjC/pull/1410#discussion_r88534964) [](ancestors = 88534964,88525013)